packages/web-runtime/tests/unit/components/SkipTo.spec.js

```diff
@@ -0,0 +1,47 @@
+import { shallowMount } from '@vue/test-utils'
+import SkipTo from '../../../src/components/SkipTo.vue'
```

Comment:
```suggestion
import SkipTo from '@files/src/components/SkipTo.vue'
```

Reply:
it should be `@runtime` but it has not been mapped.